skimage/feature/util.py

```diff
@@ -76,15 +76,24 @@ def plot_matched_features(
         The Axes object where the images and their matched features are drawn.
     keypoints_color : matplotlib color, optional
         Color for keypoint locations.
-    matches_color : matplotlib color, optional
-        Color for lines which connect keypoint matches. By default the
-        color is chosen randomly.
+    matches_color : matplotlib color or sequence thereof, optional
+        Single color or sequence of colors for each line defined by `matches`,
+        which connect keypoint matches. See [1]_ for an overview of supported
+        color formats. By default, colors are picked randomly.
     only_matches : bool, optional
         Set to True to plot matches only and not the keypoint locations.
     alignment : {'horizontal', 'vertical'}, optional
         Whether to show the two images side by side (`'horizontal'`), or one above
         the other (`'vertical'`).
 
+    References
+    ----------
+    .. [1] https://matplotlib.org/stable/users/explain/colors/colors.html#specifying-colors
+
+    Notes
+    -----
+    To make a color palette work for any number of `matches`, you can wrap the palette
```

Comment:
```suggestion
    To make a color sequence work for any number of `matches`, you can wrap that sequence
```
@lagru @stefanv what do you think?

Reply:
Done in https://github.com/scikit-image/scikit-image/pull/7541/commits/c6be3eb322115ecb743559aa4e45d5a5e45e92fa.

Reply:
Perfect.